static/js/zamboni/devhub.js

```diff
@@ -990,7 +990,10 @@ function initCatFields(delegate) {
             $("input", $misc).prop("checked", checkedLength <= 0);
         };
         var checkOther = function() {
-            $("input", $main).prop("checked", false).prop("disabled", false);
+          // Save Android categories
+          var $androidCat = $main.find('input[name="form-1-categories"]');
+          $("input", $main).prop("checked", false).prop("disabled", false);
+          $($androidCat+":checked").prop("checked", true).prop("disabled", false);
```

Comment:
This line looks suspect, trying similar concatenation on jsbin causes sizzle to blow up.

Running locally shows this causes a JS error:
```
Error: Syntax error, unrecognized expression: [object Object]:checked
```